webapp/i18n/en.json

```diff
@@ -24,13 +24,8 @@
   "CommentsList.send": "Send",
   "ContentBlock.Delete": "Delete",
   "ContentBlock.DeleteAction": "delete",
-  "ContentBlock.Text": "Text",
-  "ContentBlock.addDivider": "add divider",
+  "ContentBlock.addElement": "add {type}",
```

Comment:
I think we can just leave this as part of the content type definition, clearly i can see here "Add text" or "upload image" or "embed link"...

Reply:
Need to clean up the i18n later. Right now I'm passing in the intl object to the content handler. Not sure if there's a better way (e.g. construct one in the handler). Also looks like `formatjs extract` is not handling the formatMessage calls in the handlers.